lib/screens/create_region_screens/viewmodels/create_region_bloc.dart

```diff
@@ -14,9 +17,26 @@ class CreateRegionBloc extends Bloc<CreateRegionEvent, CreateRegionState> {
     on<OnNextTapped>(_onNextTapped);
     on<OnCreateRegionTapped>(_onCreateRegionTapped);
     on<OnBackPressed>(_onBackPressed);
+    on<UploadImage>(_uploadImage);
     on<ClearCreateRegionPageCommand>((_, emit) => emit(state.copyWith()));
   }
 
+  Future<void> _uploadImage(UploadImage event, Emitter<CreateRegionState> emit) async {
+    try {
+      final image = await ImagePicker().pickImage(source: ImageSource.gallery, imageQuality: 50, maxWidth: 2000);
+      if (image != null) {
+        final croppedFile = await ImageCropper().cropImage(
+          sourcePath: image.path,
+          aspectRatioPresets: [CropAspectRatioPreset.square],
```

Comment:
CropAspectRatioPreset.square ? 
Dont we need a rectangle here?

Reply:
It would be good to ask Jason but the placeholder is a dotted square with rounded corners